pages/common/ned.md

```diff
@@ -0,0 +1,35 @@
+# ned
+
+> Is like grep, but with powerful replace capabilities, and unlike sed, as it isn't restricted to line oriented editing.
+
+- Recursively search starting in the current directory, ignoring case:
+
+`ned -iR '^[dl]og' .`
+
+- Search always showing colored output:
+
+`ned -c '^[dl]og' .`
+
+- Search never showing colored output:
+
+`ned --colors=never '^[dl]og' .`
+
+- Set default arguments in your terminal environment:
```

Comment:
I'm not sure we need to include this example, as it's not directly related to `ned` itself (it's showing other functionality of the terminal that's documented in other pages).

What do you think, @agnivade / @pxgamer?

Reply:
Agree.

Reply:
Are you referring to the subsequent line? The NED_DEFAULTS environment variable is a feature of `ned` itself.

`export NED_DEFAULTS='-i --colors=always'`

Reply:
I don't know if this example is really needed or helpful, afaik the aim of tldr pages is not to provide a complete and detailed documentation of a command, but only the most important parts in a concise yet understandable manner (think of what's needed for a new user in order to use this command). This example looks to me more like a configuration thing that could be explained in a `man` page.

PS: I'm not a contributor (although I'm looking forward to providing some help in the future), I'm just sharing my opinion ;)

Reply:
Fair enough. Cheers!

Reply:
Done.

Reply:
You've got it, @mttbernardini!